src/axom/sidre/docs/sphinx/parallel_io_concepts.rst

```diff
@@ -10,68 +10,67 @@ Parallel File I/O
-an interface to manage parallel I/O of the data managed by Sidre.  It
-enables the writing of data from parallel runs and can
-be used for the purposes of restart or visualization.
+The Sidre ``IOManager`` class provides an interface to manage parallel I/O of 
+hierarchical data managed by Sidre. It enables the writing of data from 
+parallel runs and can be used for the purposes of restart or visualization.
 
 Introduction
 -------------
 
-The IOManager class provides parallel I/O services to Sidre.
-IOManager relies on the fact that Sidre's Group and View objects are
-capable of saving and loading themselves.  These I/O operations in Sidre are
-inherently serial, so IOManager coordinates the I/O operations of multiple
-Sidre objects that exist across the MPI ranks of a parallel run.
+The ``IOManager`` class provides parallel I/O services to Sidre. It relies on 
+the fact that Sidre ``Group`` and ``View`` objects are capable of saving and 
+loading themselves. The I/O operations provided by those two classes are 
+inherently serial. So the ``IOManager`` class coordinates I/O operations 
+of multiple Sidre objects that exist across the MPI ranks of a parallel run.
 
 * The internal details of the I/O of individual Sidre objects are opaque to
-  IOManager, which needs only to make calls to the I/O methods in Sidre's public
-  API.
-* Sidre data is written from M ranks to N files (M >= N), and the files can
+  the ``IOManager`` class, which needs only to make calls to I/O methods in 
+  ``Group`` and ``View`` class methods.
+* Sidre data can be written from M ranks to N files (M >= N), and the files can
   be read to restart a run on M ranks.
-* When saving output, a root file is created that contains some bookkeeping
-  data that is used to coordinate a subsequent restart read.
+* When saving output, a root file is created that contains bookkeeping
+  information that is used to coordinate a subsequent restart read.
 * The calling code can also add extra data to the root file to provide 
   metadata that gives necessary instructions to visualization tools.
 
-Parallel I/O using IOManager class
-----------------------------------
+Parallel I/O using the IOManager class
+---------------------------------------
 
-To accomplish parallel I/O, Sidre provides the IOManager class.  This class is
-instantiated with an MPI communicator and provides several overloads of the
-``write()`` and ``read()`` methods.  These methods save a Group in parallel
-to a set of files and read a Group from existing files.
-IOManager optionally uses the SCR library for scalable I/O
-management (such as using burst buffers if available).
+A Sidre ``IOManager`` object is created with an MPI communicator and provides 
+various overloads of ``write()`` and ``read()`` methods.  These methods save 
+a ``Group`` object in parallel to a set of files and read a ``Group`` object
+from existing files. The I/O manager can optionally use the SCR library for 
```

Comment:
Should we add a link to SCR?

Perhaps one of: 
* https://computing.llnl.gov/projects/scalable-checkpoint-restart-for-mpi
* https://github.com/LLNL/scr

Reply:
done.